pkgs/development/python-modules/moderngl/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, libGLU_combined
+, libX11
+}:
+
+buildPythonPackage rec {
+  pname = "moderngl";
+  version = "5.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0x8xblc3zybp7jw9cscpm4r5pmmilj9l4yi1rkxyf0y80kchlxq4";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ libGLU_combined libX11 ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/cprogrammer1994/ModernGL;
+    description = "ModernGL: High performance rendering for Python 3";
```

```suggestion
    description = "High performance rendering for Python 3";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.